compiler/testData/codegen/box/delegatedProperty/delegateToConst/delegateToConst.kt

```diff
@@ -0,0 +1,13 @@
+// WITH_STDLIB
```

Comment:
Please split the tests so that the main behavior part is tested here, and the part which uses reflection (`getDelegate`) is tested in `compiler/testData/codegen/box/reflection/properties/getDelegate/method`. It's needed because the main part should pass on all backends (JVM, JS, Native, WASM), but reflection is only supported on JVM at the moment. So the reflection part should also have the directive `// TARGET_BACKEND: JVM`.

Also, `WITH_STDLIB` is not needed here because it's implied by `WITH_REFLECT`.

Reply:
@strangepleasures Please move tests on reflection to `compiler/testData/codegen/box/reflection/properties/getDelegate/method`